app/models/GutterTests.scala

```diff
@@ -0,0 +1,51 @@
+package models
+
+import io.circe.generic.extras.Configuration
+import io.circe.generic.extras.auto._
+import io.circe.generic.extras.semiauto._
+import io.circe.{Decoder, Encoder, Json}
+import models.Methodology.defaultMethodologies
+
+case class GutterContent(
+    image: Option[Image],
+    bodyCopy: Option[List[String]],
+    cta: Option[Cta],
+)
+
+case class GutterVariant(
+    name: String,
+    gutterContent: Option[GutterContent],
+)
+
+case class GutterTest( // TODO: remove inappropriate ones.
```

Comment:
shall we do this now?

Reply:
Yep, sorry that's old - just searched for TODOs and there are a few out-of-date ones.